iris/service/rest/queries.py

```diff
@@ -79,6 +79,23 @@ def genericTermsQuery(value):
     }
 
 
+def rangeFilter(field, operators=('gte', 'lt')):
+    def handleFilter(value):
+        operations = {}
+        for i, op in enumerate(operators):
+            if i > len(value):
+                break
+            v = value[i]
```

Comment:
in the next line, you're making sure that "v" is "truthy". can ``value[i]`` every raise a ``KeyError``?
If so and if you do not want to fail hard here, you might want to consider changing this to ``value.get(i, None)``